get-version.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash -ex
 
-tag=$(git tag --points-at HEAD | grep -E '^[0-9]{8}(-[0-9]+)?$' |
+branch=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
Can we validate that? To make sure that it does not do any weird regex and we do not understand why it is doing something unexpected. Like limiting to `[a-z0-9/]+`.